shortstories/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+class ShortStory(models.Model):
+     PUBLICATION_STATUS =(
+        ('dr', 'Draft'),
+        ('p' , 'Published'),
+        ('b' , 'Banned'),
+        ('de', 'Deleted'),
+    )
+    user = models.Foreignkey(settings.AUTH_USER_ MODEL)
```

Comment:
you should put this user inside the author, there is no need to have another field called user. Also you can't use `settings.    something` without actually importing the settings in your python file

Reply:
author = models.Foreignkey(user,on_delete = models.CASCADE) , so this is right? just need to import user? 
but links you sent to me for importing user was like defining user like this? !

Reply:
it is not clear for me

Reply:
Yes, you need to pay attention to the task definition which asked you to only use `author`, since you haven't imported it before and directly paste `user` in the foreign key I need to show you a help link. I barely can find a link which uses a field called a random name such as `author` and luckily import the `user` into it.